internal/app/commands/snapshot/restore.go

```diff
@@ -24,11 +24,13 @@ var Restore = cli.Command{
 			Name:  "snapshot, snap",
 			Usage: "The name of the snapshot to restore",
 		},
+		cli.BoolFlag{
+			Name:  "latest",
+			Usage: "The name of the snapshot to restore",
```

Comment:
Copy/paste? Should be different description.

Reply:
Fixed !